signalfx-org-metrics/README.md.jinja

```diff
@@ -67,21 +67,21 @@ These metrics track limits that Infrastructure Monitoring enforces for your orga
 * `sf.org.limit.hosts` (gauge): Maximum number of hosts that can send data to your organization.
    The limit is higher than your contractual limit to allow for burst and overage usage. If you
    exceed this limit, Infrastructure Monitoring drops data points from new hosts but keeps
-   data points for existing hosts. To monitor your usage against the limit, use the metric
+   accepting data points for existing hosts. To monitor your usage against the limit, use the metric
    `sf.org.numResourcesMonitored` and filter for the dimension `resourceId:hosts`.
 
 * `sf.org.limit.containers` (gauge): Maximum number of containers that can send data to your
-   organization. This limit is higher than your contractual limit to give room for burst and overage usage.
+   organization. This limit is higher than your contractual limit to allow for burst and overage usage.
    If you exceed this limit, Infrastructure Monitoring drops datapoints from new containers but keeps
-   data points for existing containers. To monitor your usage against the limit, use the metric
+   accepting data points for existing containers. To monitor your usage against the limit, use the metric
    `sf.org.numResourcesMonitored` and filter for the dimension `resourceId:containers`.
 
-* `sf.org.limit.customMetricTimeSeries` (gauge): Maximum number of active custom MTS,
-   within a moving window of the previous 60 minutes, that can send data to your organization.
-   If you exceed this limit, Infrastructure Monitoring drops data points for the custom MTS
-   that exceeded the limit, but it continues to accept data points for custom MTS that already existed.
-   you’ve defined, use the metric `sf.org.numCustomMetrics`. To learn more about custom MTS, see the
-   section [About custom, bundled, and high-resolution metrics](https://docs.signalfx.com/en/latest/admin-guide/usage.html)
+* `sf.org.limit.customMetricTimeSeries` (gauge): Maximum number of active custom MTS, within a moving window of the
```

Comment:
style guide: spell out first reference of any acronyms :)

Reply:
Good point, but you're not seeing all of the text in the file. I think I spell out MTS at first usage.